mergin/client.py

```diff
@@ -95,27 +99,54 @@ def save_project_file(project_directory, data):
     with open(project_file, 'w') as f:
         json.dump(data, f, indent=2)
 
+def decode_token_data(token):
+    if not token.startswith("Bearer ."):
+        raise ValueError("Invalid token type")
+
+    try:
+        data = token[8:].split('.')[0]
```

Comment:
Add a comment or save "Bearer ." length (guessing) as a constant to avoid kind of random numbers in code and to make it more readable.